Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneFacade/SceneLODInfoExtensions.cs

```diff
@@ -14,7 +14,10 @@ public static class SceneLODInfoExtensions
             ILODCache lodCache,
                                                                 World world)
         {
-            lodCache.Release(sceneLODInfo.id, sceneLODInfo.metadata);
+            //Only try to release SceneLODInfo that has been initialized
+            if (!string.IsNullOrEmpty(sceneLODInfo.id))
```

Comment:
Can we introduce a method or an extensions `IsInitialized` instead to not repeat it every time?